src/main/scala/com/fortysevendeg/translatebubble/ui/components/ActionsView.scala

```diff
@@ -31,55 +31,52 @@ import com.fortysevendeg.translatebubble.R
 import com.fortysevendeg.translatebubble.ui.commons.Dimens
 import com.fortysevendeg.translatebubble.ui.components.DisableViewTweaks._
 import macroid.FullDsl._
-import macroid.{ContextWrapper, ServiceContextWrapper, Snail, Tweak}
+import macroid._
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Promise
 import scala.util.Success
 
 class ActionsView(context: Context, attrs: AttributeSet, defStyleAttr: Int)(implicit contextWrapper: ServiceContextWrapper)
-  extends FrameLayout(context, attrs, defStyleAttr)
-  with ActionViewLayout {
+    extends FrameLayout(context, attrs, defStyleAttr)
+    with ActionViewLayout {
 
   def this(context: Context)(implicit contextWrapper: ServiceContextWrapper) = this(context, null, 0)
 
   def this(context: Context, attr: AttributeSet)(implicit contextWrapper: ServiceContextWrapper) = this(context, attr, 0)
 
-  val sizeClose = contextWrapper.application.getResources.getDimension(R.dimen.size_close).toInt
+  val resources = contextWrapper.application.getResources
 
-  val marginClose = contextWrapper.application.getResources.getDimension(R.dimen.margin_close).toInt
+  val sizeClose = resources.getDimension(R.dimen.size_close).toInt
 
-  val sizeDisable = contextWrapper.application.getResources.getDimension(R.dimen.size_disable).toInt
+  val marginClose = resources.getDimension(R.dimen.margin_close).toInt
 
-  val marginTopDisable = contextWrapper.application.getResources.getDimension(R.dimen.margin_top_disable).toInt
+  val sizeDisable = resources.getDimension(R.dimen.size_disable).toInt
 
-  val marginSeparateOptionsDisable = contextWrapper.application.getResources.getDimension(R.dimen.margin_separate_options_disable).toInt
+  val marginTopDisable = resources.getDimension(R.dimen.margin_top_disable).toInt
+
+  val marginSeparateOptionsDisable = resources.getDimension(R.dimen.margin_separate_options_disable).toInt
```

Comment:
Instead of create a `val` for `resources` what about a `def` to save the `toInt` method also?:
```scala
def intDimension(id: Int) = contextWrapper.application.getResources.getDimension(id).toInt

// val sizeDisable = resources.getDimension(R.dimen.size_disable).toInt
val sizeDisable = intDimension(R.dimen.margin_close)
```